src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,11 +1,22 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private String baseOfBulldozer;
+
+    public Bulldozer(String name, String baseOfBulldozer, String color) {
```

Comment:
what about other parameters `name` and `color` why we nit use them?

Reply:
Because It is in Machine class and Buldozer extends Machine. I added only some type-specific fields to Buldozer classes.